Formula/pmd-emacs.rb

```diff
@@ -0,0 +1,30 @@
+require File.expand_path("../../Homebrew/emacs_formula", __FILE__)
+
+class PmdEmacs < EmacsFormula
+  desc "Major mode for plugging PMD into Emacs"
+  homepage "https://github.com/pmd/pmd-emacs"
+  url "https://github.com/pmd/pmd-emacs/archive/0.8.tar.gz"
+  sha256 "d4677bf1039f2a25b0a029c593bfc1f442c8fb0807b17092ce422cd25f390f42"
+  head "https://github.com/pmd/pmd-emacs.git"
+
+  depends_on :emacs => "22.1"
+  depends_on "pmd"
+
+  def install
+    # install the documentation
+    doc.install "CHANGELOG", "LICENSE.txt", "readme.md"
```

Comment:
Just tested again, and this line is indeed necessary. Without it, none of the 3 files listed are installed.

Not sure if this is a problem with Linuxbrew (if you could test this on a Mac, it would be greatly appreciated!), with Emacs formulae in particular, or some other combination.